proxy/http/PreWarmManager.h

```diff
@@ -312,6 +216,16 @@ class PreWarmQueue : public Continuation
 class PreWarmManager
 {
 public:
+  PreWarmManager()
+  {
+    // We use the callback because it would introduce a circular dependency
+    // for SNIConfig to explicitly call prewarmManager.reconfigure().
+    // Because prewarmManager is global there's not a good place to set
+    // this, but as long as there's only one instance of this class
+    // there should not be an issue.
+    SNIConfig::set_on_reconfigure_callback([this]() { this->reconfigure(); });
```

Comment:
🥇

Reply:
Nice idea to get rid of the cyclic dependency 👍